### PR TITLE
[READY] Adding Helmsman container image

### PR DESCRIPTION
**Pull request by Lynty**
Pending Helmsman nixpkg expression distributed to unstable channel, publish to Nebulaworks dockerhub, and testing.

- [x] Created helmsman [derivation](https://github.com/NixOS/nixpkgs/pull/86328)
- [x] Created pin with iteration set to `0` for nixpkgs-unstable channel
- [x] Published `nebulaworks/helmsman-aws` to [Nebulaworks org in Docker Hub](https://hub.docker.com/repository/docker/nebulaworks/helmsman-aws)
- [x] Tested image:
```
[ldong@MBP]$ docker pull nebulaworks/helmsman-aws:72rs42ar65syxrgq6w0qyv98n97zkr6n
[ldong@MBP]$ docker exec -it <container_hash> bash

bash-4.4# jq
jq - commandline JSON processor [version 1.6]

Usage:  jq [options] <jq filter> [file...]
        jq [options] --args <jq filter> [strings...]
        jq [options] --jsonargs <jq filter> [JSON_TEXTS...]

jq is a tool for processing JSON inputs, applying the given filter to
its JSON text inputs and producing the filter's results as JSON on
standard output.

The simplest filter is ., which copies jq's input to its output
unmodified (except for formatting, but note that IEEE754 is used
for number representation internally, with all that that implies).

For more advanced filters see the jq(1) manpage ("man jq")
and/or https://stedolan.github.io/jq

Example:

        $ echo '{"foo": 0}' | jq .
        {
                "foo": 0
        }

For a listing of options, use jq --help.

bash-4.4# helmsman

 _          _
| |        | |
| |__   ___| |_ __ ___  ___ _ __ ___   __ _ _ __
| '_ \ / _ \ | '_ ` _ \/ __| '_ ` _ \ / _` | '_ \
| | | |  __/ | | | | | \__ \ | | | | | (_| | | | |
|_| |_|\___|_|_| |_| |_|___/_| |_| |_|\__,_|_| |_| version: v3.3.0
A Helm-Charts-as-Code tool.

2020-05-19 21:38:35 INFO: cmd.Start: exec: "helm": executable file not found in $PATH
2020-05-19 21:38:35 CRITICAL: While checking helm version: exec: "helm": executable file not found in $PATH

bash-4.4# aws
usage: aws [options] <command> <subcommand> [<subcommand> ...] [parameters]
To see help text, you can run:

  aws help
  aws <command> help
  aws <command> <subcommand> help
aws: error: the following arguments are required: command

bash-4.4# curl
bash: curl: command not found
```
- [x] successful helmsman [job in gitlab pipeline](https://gitlab.com/nebulaworks/eng/infra/-/jobs/568410581)

### CODE REVIEW

File: imgs/helmsman/default.nix [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{ system ? builtins.currentSystem }:
```
Comment (sarcasticadmin) [inline imgs/helmsman/default.nix:1]: Can we have the dir match the container name? `helmsman-aws` in this case?

File: pin/pins/nixpkgs-unstable_0.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (sarcasticadmin) [inline pin/pins/nixpkgs-unstable_0.json:1]: I didnt realize there was both a `nixpkgs-unstable` and ` nixos-unstable` branch in `nix-channels` 🤦 Now I know and this looks good!

File: imgs/helmsman-aws/default.nix [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  # when publishing to registry
+  tag = "latest";
+  extraCommands = ''
+      # make sure /tmp exists
+      mkdir -m 1777 tmp
+  '';
+  config = {
+    Env = [ 
+      "PATH=/bin/"
+      "HELM_PLUGINS=/share/helm/plugins/"
```
Comment (Lynty) [inline imgs/helmsman-aws/default.nix:27]: `shellhook` didn't work, using this for now; I will investigate `shellhook` and `makeWrapper` when time permits

**Comment by sarcasticadmin** [comment]
@Lynty awesome, thanks for all the hardwork!

Confirmed this looks good in dockerhub too:
```
$ skopeo inspect docker://docker.io/nebulaworks/helmsman-aws@sha256:2cab619b7e43b30eb8384b9104b19499e4b0ac39c4347ae77ec5b3c65007c103
{
    "Name": "docker.io/nebulaworks/helmsman-aws",
    "Digest": "sha256:2cab619b7e43b30eb8384b9104b19499e4b0ac39c4347ae77ec5b3c65007c103",
    "RepoTags": [
        "0r4rskk3b5han0zzncj67sjhkm7lip1w",
        "2vil2iz4gz0lmcqsg92kxylhh4mds6jv",
        "72rs42ar65syxrgq6w0qyv98n97zkr6n",
        "7ar2sigpxxyll06yvgan428qn74k0vqm",
        "7hl8p275d7f3gcnam52s7hg1d307a2ik",
        "axk5j5bhv0sv9ys1xj4bmhc34kfw5jkn",
        "ibx9pd0yzvspf6gczmdmmhmam9sr25wr",
        "sb47bkwd854ws9dc4gzsvnln06pidxjf",
        "xym39zji91ga5rbw6p8xbwy5dcbizbn3"
    ],
    "Created": "1970-01-01T00:00:01Z",
    "DockerVersion": "",
    "Labels": {
        "com.nebulaworks.packages": "awscli:1.17.13,bash:4.4,coreutils:8.31,helm-diff:3.1.1,helm:3.2.1,helmsman:3.3.0,jq:1.6,kubectl:1.18.1,nss-cacert:3.52",
        "org.opencontainers.image.authors": "Nebulaworks Inc.",
        "org.opencontainers.image.source": "https://github.com/Nebulaworks/nix-garage"
    },
    "Architecture": "amd64",
    "Os": "linux",
    "Layers": [
        "sha256:b59069cadaedf5c0ebef846dcc248bb64a9ee16b972fe6c8d6769efec56a1795"
    ]
}
```

**Pull request merged**